css/index.css

```diff
@@ -126,6 +158,50 @@ p {
     padding-right: 0;
 }
 
+
+/* container about-page */
+main {
+    /* border: 2px solid darkblue; */
```

Comment:
Awesome, I love how you place comments above some of the classes like in like 162. That being said, in line 164 you have commented out code. If you don't need it, just delete it :)